### PR TITLE
Add rockspec

### LINKED ISSUE #5: Add LuaRock spec
please create spec to allow to install Lua module using LuaRocks [1]

1. https://luarocks.org/

### PR DESCRIPTION
**Pull request by ligurio**
Closes #5

How to test:
```
$ luarocks --local make cparser-scm-1.rockspec 
$ luarocks --local remove cparser
```

### CODE REVIEW

**Comment by leonbottou** [comment]
I merged the pull request.It builds locally.

I had trouble uploading it. For a long time, trying to install it gave
```
% luarocks install --server=https://luarocks.org/dev cparser

Installing https://luarocks.org/dev/cparser-scm-1.rockspec

Error: Couldn't extract archive /var/folders/y5/x36j2q_d6d9_qd69ny6yk43r01468q/T//luarocks_luarocks-source-cparser-scm-1-nGubVH/CParser: unrecognized filename extension
```

I tweaked things and it now seems to work. I do not know whether my tweaks did it, or whether there is a delay.

**Comment by ligurio** [comment]
Thanks!

File: cparser-scm-1.rockspec [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package = 'cparser'
+version = 'scm-1'
+source  = {
+    url    = 'https://github.com/facebookresearch/CParser',
```
Comment (Totktonada) [inline cparser-scm-1.rockspec:4]: `https://` is interpreted as for tarball downloading. Use `git+https://` instead.

See `source.url` description here: https://github.com/luarocks/luarocks/wiki/Rockspec-format#build-rules

Comment (lb3361) [reply cparser-scm-1.rockspec:4]: The main version uses 
```
    url    = 'git://github.com/facebookresearch/cparser',
```
which hopefully works already.

**Pull request merged**